dfm-sideline-sidekick-app/ConditionsSection.tsx

```diff
@@ -0,0 +1,257 @@
+                structures, so fractures are more common and can be unstable. This creates risk for
+                potential damage to the spinal cord resulting in quadriplegia and death and could be
+                made worse by moving patients without proper immobilization
+              </Text>
+            </View>
+
+            <View style={styles.infoSection}>
+              <Text style={styles.descriptionTitle}>Mechanism of Injury</Text>
+              <BulletList
+                items={[
+                  "Direct blow to head/neck",
+                  "Axial loading to spine, esp. w/neck in flexion",
+                ]}
+              />
+            </View>
+
+            <View style={styles.infoSection}>
+              <Text style={styles.descriptionTitle}>Diagnosis</Text>
+              <BulletList items={["Local pain", "Ecchymosis, and swelling"]} />
+            </View>
+
+            <View style={styles.infoSection}>
+              <Text style={styles.descriptionTitle}>Physical Exam</Text>
+              <BulletList items={["TTP over spinous process or vertebral bodies"]} />
+            </View>
+          </View>
+
+          <View style={isTreatmentPressed ? styles.howToTreat : styles.howToTreatHidden}>
+            <View style={styles.infoSection}>
+              <Text style={styles.descriptionTitle}>Acute Management</Text>
+              <BulletList
+                items={[
+                  "Immobilize with spine board, cervical-collar, and barriers to lateral head movement (or whole body vacuum splint)",
+                  "If this is not available, immobilize by placing hands on patient shoulders and using forearms to immobilize head",
+                ]}
+              />
+            </View>
+
+            <View style={styles.infoSection}>
+              <Text style={styles.descriptionTitle}>Dispo</Text>
+              <BulletList items={["Emergency transport to ED for CT (most accurate) +/- XR"]} />
+            </View>
+
+            <View style={styles.infoSection}>
+              <Text style={styles.descriptionTitle}>Considerations</Text>
+              <Text style={styles.descriptionInfo}>
+                If any suspicion for injury, send to ED. However, less likely if the following
+                criteria are met:
+              </Text>
+              <BulletList
+                items={[
+                  "No cervical spine tenderness",
+                  "Normal alertness/consciousness/GCS 15",
+                  "No major distracting injuries",
+                  "Normal neurologic status (full strength/sensation in all extremities)",
+                  "Ability to actively rotate neck to 45 degrees laterally in both directions",
+                ]}
+              />
+            </View>
+          </View>
+        </View>
+      </ScrollView>
+    </SafeAreaView>
+  );
+}
+
+const styles = StyleSheet.create({
```

Comment:
Make a separate file called "ConditionSectionStyles.tsx" or something along those lines that contains the styles

Reply:
If you import styles from the file at the top of your file, this will fix many of the linter errors